### ISSUE TITLE
hardcoding server value

### ISSUE THREAD

**Issue by ReshmaJaiswal** [H2-invent/jitsi-admin#520]
I want to hardcode server value instead of allowing everyone to add server manually , i am using laf_CreateServerOpenToEveryone=0 then i need to add atleast 1  server so where i do changes so that it take only 1 server everytime without adding manually or showing option to add

**Comment by holema** [comment]
Hello,

To configure this, you can set the variable `laf_CreateServerOpenToEveryone` to 0. This parameter contains a Keycloak group whose members are allowed to set up a server: `laf_keycloakGroupToCreateServer=/user`. Here, for instance, you can set `/admin` and then create a Keycloak group named "admin". Add the users to this group whom you wish to allow to create a server.

Best regards,

**Comment by ReshmaJaiswal** [comment]
I dont want anyone to allow i want to hardcode for everyone

**Comment by holema** [comment]
this you can do by set the default server id in the env: `default_jitsi_server_id=0` The id is the id in the database

**Comment by ReshmaJaiswal** [comment]
is there any way to set max participant limit?

**Comment by holema** [comment]
not now. You can set a limit to alow a number of self invitiation for the webinar mode but not a hard server limit.

**Comment by ReshmaJaiswal** [comment]
how can i limit for server

**Comment by holema** [comment]
hello @ReshmaJaiswal, you want to limit the users which are in one conference?

**Comment by ReshmaJaiswal** [comment]
yes.suppose if i added 1 confernce so it must add only 80 partcipant

**Comment by ReshmaJaiswal** [comment]
i want to limit participant in my confernce

**Comment by ReshmaJaiswal** [comment]
if i want in 1 meet there should be only 100 participant then more than this could not join meet

**Comment by ReshmaJaiswal** [comment]
?

**Comment by holema** [comment]
Up to now this is ont possible and implemented. It would be possible to implemtn this when the event sync of jitsi is installed on jitsi side. When we install the jitsi and jitsi-admin stack the nwe install the event sync plugin but this is not that easy. The funvtion to limit the number of participants is a feature request.

**Comment by ReshmaJaiswal** [comment]
ok
